hyperopt/suggest.py

```diff
@@ -0,0 +1,122 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from future import standard_library
+from builtins import object
+import logging
+import os
+
+import numpy as np
+
+from .utils import coarse_utcnow
+from . import base
+from . import tpe
+from . import hp
+
+from pandas import DataFrame
+from datetime import datetime
+
+
+class SuggestObject(object):
+    """Object for conducting search experiments.
+    """
+
+    def __init__(self, algo, domain, trials, rstate):
+        self.algo = algo
+        self.domain = domain
+        self.trials = trials
+        self.rstate = rstate
+
+    def serial_evaluate(self):
+        for trial in self.trials._dynamic_trials:
+            if trial['state'] == base.JOB_STATE_NEW:
```

Comment:
What does this mean? Is this line relevant for our purpose? Can we further simplify the code?

Reply:
removed